### ISSUE TITLE
All websites dashboard is missing tag <h1> invalid for accessibility

### ISSUE THREAD

**Issue by tassoman** [piwik/piwik#20869]
Multisites Index controller is missing WCAG 2.1 requirement 1.3.1

Page's title have tag h2 instead of h1

## Context

Multisite Plugin

## Expected Behavior

Every page must have at least heading `<h1>`

## Current Behavior

```html
<h2 class="card-title">
<div class="enrichedHeadline">
<div class="title" tabindex="6">All Websites dashboard <span class="smallTitle" title="29 visits in yesterday compared to 7 in 2023-06-07. Evolution: +314}%">(Total: <strong>29</strong> visits, <strong>208</strong> pageviews, <strong>212</strong> actions, <strong>0</strong> revenue)</span></div><!----><span class="iconsBar" style="display: none;"><a rel="noreferrer noopener" target="_blank" class="helpIcon" href="https://matomo.org/faq/new-to-piwik/all-websites-dashboard/" title="Help (opens in new tab)"><span class="icon-help"></span></a><!----><div class="ratingIcons"><!----></div></span><div class="inlineHelp" style="display: none;"><div></div><span class="helpDate"></span><a rel="noreferrer noopener" target="_blank" class="readMore" href="https://matomo.org/faq/new-to-piwik/all-websites-dashboard/">More Details</a></div></div></h2>
```

## Possible Solution

```html
<div class="card-title">
<div class="enrichedHeadline">
<h1 class="title" tabindex="6">All Websites dashboard <span class="smallTitle" title="29 visits in yesterday compared to 7 in 2023-06-07. Evolution: +314}%">(Total: <strong>29</strong> visits, <strong>208</strong> pageviews, <strong>212</strong> actions, <strong>0</strong> revenue)</span>
```
etc...

## Steps to Reproduce (for Bugs)
Inspect this page, using browser's tool
https://demo.matomo.cloud/index.php?module=MultiSites&action=index&idSite=1&period=day&date=yesterday

## Your Environment
* Matomo Version: 4.x
* PHP Version: 8.x
* Server Operating System: Centos7
* Browser: any

**Comment by bx80** [comment]
Thanks for the feedback @tassoman, I've assigned this issue for prioritization :+1:

**Comment by ronak-innocraft** [comment]
Will be released as part of 5.3.0
